### PR TITLE
add webpack to build process

### PR DESCRIPTION
**Pull request by vbence86**

### CODE REVIEW

**Comment by jbalsas** [comment]
Hey @vbence86, since we're at it, maybe we can tackle https://github.com/metal/metal-clipboard/issues/2 😉

File: package.json [hunk 1]
```diff
@@ -25,15 +26,18 @@
...
   "dependencies": {
+    "bootstrap": "^3.3.6",
     "metal": "^2.0.0",
     "metal-state": "^2.0.0",
     "metal-dom": "^2.0.0"
   },
   "devDependencies": {
     "babel-cli": "^6.4.5",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-transform-node-env-inline": "^0.2.0",
```
Comment (jbalsas) [inline package.json]: Do we need this one? If we don't have incremental-dom I think this is not needed.

Comment (vbence86) [reply package.json]: Yes, you are right, I'm addressing this now.

**Comment by vbence86** [comment]
I'd tried to extract `ClipboardAction` into a self-contained component, but I stumbled across an obvious circular dependency issue, which may have been the reason it was implemented into `Clipboard.js` in the first place, and it didn't allow me to copy the content of it into a separate file without the need of further refactoring. With all this in mind, I just add the `ClipboardAction` to the export keyword.

**Pull request merged**